www-src/app/replicator/replicator_config.coffee

```diff
@@ -37,14 +50,21 @@ module.exports = class ReplicatorConfig extends Backbone.Model
                 @remote = @createRemotePouchInstance()
                 callback? null, this
 
-    getScheme: () ->
+        options = @makeDSUrl('/filters/config')
+        options.body = @getConfigFilter()
+        request.put options, (err, res, body) =>
+            return callback err if err
+            return callback body unless body.success or body._id
+            callback? null, this
```

Comment:
It's weird that `callback` is mandatory on the first two cases but optional in the third.